app/serializers/tutorial_serializer.rb

```diff
@@ -4,7 +4,12 @@
 require 'user_serializer'
 
 class TutorialSerializer < ActiveModel::Serializer
-  attributes :id, :meeting_day, :meeting_time, :meeting_location, :abbreviation, :campus_id, :capacity, :tutor_name, :num_students
+  attributes :id, :meeting_day, :meeting_time, :meeting_location, :abbreviation, :campus_id, :capacity, :tutor_name, :num_students,
```

Comment:
We can remove tutor name from here... we should be able to link to this if we have tutor id or something similar.